docs/lifecycle/lifecycle.rst

```diff
@@ -117,36 +129,129 @@ Board LFN Entry Review
 ++++++++++++++++++++++
 
 It is up to the Board to define its own criteria and process of the Board's LFN
-Entry Review. The TAC recommends the Board make its LFN Entry Review criteria
-and process public and accept design input from the public.
+Entry Review. Such review may include legal, trademark, and license reviews. The 
+TAC recommends the Board make its LFN Entry Review criteria and process public 
+and accept design input from the public.  
+
+Budget Guidance: The TAC recommends that the governing board allocate no funding 
+to Sandbox projects and that those projects cannot use cross-project funding
+buckets.
```

Comment:
OK with that
we may maybe just imagine some indirect funding such as invitation to summits to introduce a new project

Reply:
yes, and things like having their logo on marketing materials.  Open to suggestions on wording to that effect.

Reply:
updated in next commit